server/server.go

```diff
@@ -194,6 +200,11 @@ func (s *Server) SetSealer(sealer Sealer) {
 	s.sealer = sealer
 }
 
+// RegisterRPC publishes in the server the methods on rcvr.
+func (s *Server) RegisterRPC(rcvr interface{}) error {
+	return s.dispatcher.Register(rcvr)
+}
```

Comment:
Right, but my point is, would this be called by somebody setting up a server (e.g., what we've got in `cmd/gokeyless`), or by an actual client (e.g., in the standard Cloudflare edge/origin server running Keyless setup, called by the Cloudflare edge)?